### PR TITLE
Fix s3-fuzzer

### PR DESCRIPTION
**Pull request by tmacro**
Refactor and fix nightly s3 listing fuzzer run

### CODE REVIEW

File: tests/Makefile [hunk 1]
```diff
@@ -241,17 +241,20 @@ run-listing-fuzzer:
 		--rm \
 		--restart=Never \
 		--attach=True \
-		--image=zenko/s3fuzz:latest \
+		--image=tmacro/s3fuzz:latest \
```
Comment (bennettbuchanan) [inline tests/Makefile]: Do we want to pull from your personal docker hub image?

**Review by bennettbuchanan — APPROVED** [review]

File: tests/Makefile (hunk 1 above)
Comment (rahulreddy) [reply tests/Makefile]: That is of concern, we should be using zenko's registry

Comment (tmacro) [reply tests/Makefile]: From the slack message: `custom docker image left in purposefully until done with testing`

Comment (bennettbuchanan) [reply tests/Makefile]: Would it be possible to push a custom test image to the zenko account in docker hub instead? If we do that, the test won't be bound to your account. 

Or maybe I'm misunderstanding that you'll push another commit to change it back before this PR is merged.

Comment (tmacro) [reply tests/Makefile]: I meant the second, I'm doing some testing to make sure its deterministic and then I'll swap it over

**Review by ssalaues — APPROVED** [review]

**Comment by tmacro** [comment]
@bert-e approve

**Pull request merged**